src/path.rs

```diff
@@ -1,3 +1,5 @@
+#[allow(unused_imports)]
```

Comment:
This import is only needed when `path_try_exists` feature is enabled. Without the allow unused imports clippy generates a warning. Maybe there's a better way to avoid it. 

In this case I think it affects the whole file, not just this one import. Which is not ideal